components/leaflet-maps/ClustersMap.vue

```diff
@@ -21,7 +21,7 @@
 
               <div class="font-semibold text-base">{{ witch.name }}</div><br>
               <div>
-                <b>Investigation Date:</b> {{ witch.investigationDates[1] }}<br>
+                <b>Investigation Date:</b> {{ getFormatDate(witch.investigationDates[1],witch.precision) }}<br>
```

Comment:
Please could you set up Prettier on your machine? It will help auto-format the code. Something that stood out for me here was the lack of a whitespace after the comma. It is minor and not worth manually fixing so prettier should handle this. I believe it now has good support for Vue: https://prettier.io/

Reply:
Sorry Ruby, just checking this has been set up as I can still format stuff in this PR and Prettier changes things. Cheers

Reply:
Has been set up properly now.

Reply:
Cool, thank you